client/components/forms/docs/example.jsx

```diff
@@ -35,7 +36,10 @@ import PhoneInput from 'components/phone-input';
 /**
  * Internal dependencies
  */
-var countriesList = require( 'lib/countries-list' ).forSms();
+const countriesList = require( 'lib/countries-list' ).forSms();
+const currencies = require( 'lib/format-currency/currencies' ).CURRENCIES
```

Comment:
I would leave a note here if we need `require` here for sync vs `import` async behavior. This is currently a mostly babel implementation detail that it behaves this way.

Alternatively I'd recommend moving the two lists into a module or in the Component constructor.